### PR TITLE
Revise Docker documentation

### PR DESCRIPTION
**Pull request by oplantalech**
Quick review to the Docker documentation:
- Align all documentation to point to the same cBioPortal image (coming from Datahub).
- Remove the second method of loading studies to simplify.
- Complete the section of Uninstalling cBioPortal removing all containers and images.
- Update link to Docker documentation (contains more information).
- Remove references to $PWD and $HOME and instead ask the user to place its specific path.
- Use only one command to create the cBioPortal database.
- Refactor the last part of the Keycloak documentation by pointing to the document on how to set up cBioPortal in Docker.

### CODE REVIEW

File: docs/docker/using-keycloak.md [hunk 1]
```diff
@@ -56,10 +56,21 @@ documentation](../Authenticating-and-Authorizing-Users-via-keycloak.md#configure
...
-```
+<pre>
+docker run -d --restart=always \
+    --name=cbioportal-container \
+    --net=cbio-net \
+    -v /<path_to_config_file>/portal.properties:/cbioportal/portal.properties:ro \
+    -e JAVA_OPTS='
+        -Xms2g
+        -Xmx4g
+        <b>-Dauthenticate=saml</b>
```
Comment (inodb) [inline docs/docker/using-keycloak.md]: @oplantalech neat 👍

**Comment by inodb** [comment]
This solves: https://github.com/cBioPortal/cbioportal/issues/6852

File: docs/docker/using-keycloak.md [hunk 2]
```diff
@@ -56,10 +56,21 @@ documentation](../Authenticating-and-Authorizing-Users-via-keycloak.md#configure
...
+the `JAVA_OPTS` argument in Step 6 to include the `-Dauthenticate=saml` flag:
 
-```
-    -e JAVA_OPTS='-Xms2g -Xmx4g -Dauthenticate=saml' \
-```
+<pre>
+docker run -d --restart=always \
+    --name=cbioportal-container \
+    --net=cbio-net \
+    -v /<path_to_config_file>/portal.properties:/cbioportal/portal.properties:ro \
```
Comment (inodb) [inline docs/docker/using-keycloak.md]: @oplantalech I think the ` /<path_to_config_file>/` needs to be escaped somehow - if you look at the rich diff it shows up as `//`

Comment (inodb) [reply docs/docker/using-keycloak.md]: Screen Shot 2019-12-18 at 11 20 31 AM

File: docs/docker/example_commands.md [hunk 3]
```diff
@@ -1,32 +1,32 @@
...
 docker run -it --rm --net cbio-net \
     -v /<path_to_config_file>/portal.properties:/cbioportal/portal.properties:ro \
-    -v "$PWD/study-dir:/study:ro" \
-    -v "$HOME/Desktop:/outdir" \
-    cbioportal-image \
-    metaImport.py -u http://cbioportal-container:8080 -s /study --html=/outdir/report.html
+    -v "<path_to_study_directory>:/study:ro" \
+    -v "<path_to_report_folder>:/report" \
+    cbioportal/cbioportal:latest \
+    metaImport.py -u http://cbioportal-container:8080 -s /study --html=/report/report.html
```
Comment (johnyaku) [inline docs/docker/example_commands.md]: Still struggling to import study data, even with the new image.
Attempting to import cbioportal/core/src/test/scripts/test_data/study_es_0 and datahub/public/acbc_mskcc_2015...
Validation step reports: 
> INFO: -: Unable to read xml containing cBioPortal version.
Looking at validateData.py this seems to refer to pom.xml 

Other errors include (study_es_0)...
>ERROR: meta_gsva_pvalues.txt: `geneset_def_version` is different from the geneset_version in the database; value encountered: 'msigdb_6.1'
>ERROR: data_cancer_type.txt: line 1: column 5: Unknown parent for cancer type 'brca-es0'; value encountered: 'Breast'

And for acbc_mskcc_2015...
>ERROR: -: Cancer type of study is neither known to the portal nor defined in a cancer_type file; value encountered: 'acbc'

**Pull request merged**